### DISCUSSION TITLE
Double vs Single Arrow

### DISCUSSION THREAD

**Discussion by brucegomes** [IntelliTect/EssentialCSharp.Web#14] [Feedback and Ideas] [answered]
To me a double arrow symbol doesn't make sense If I am just going to the next page. Instead, I tend to think it would skip multiple pages, or chapter, or some kind of fast forward.

![image](https://user-images.githubusercontent.com/16536254/189261189-26ae2c45-0a46-4fee-9727-0b6658d5debc.png)

Suggestion: An arrow with a single head.

![image](https://user-images.githubusercontent.com/16536254/189261410-db3ade27-d793-4e7b-8349-2567d54d6274.png)

**Comment by BenjaminMichaelis** [comment, marked as answer]
Thanks for the feedback! We have updated the styling for this!
